private-distributors-list.md

```diff
@@ -6,15 +6,18 @@ security issues.
 
 ### Embargo Policy
 
-The information members receive on distributors-announce@kubernetes.io must not be
-made public, shared, nor even hinted at anywhere beyond the need-to-know within
-your specific team except with the list's explicit approval. This holds
-true until the public disclosure date/time that was agreed upon by the list.
-Members of the list and others may not use the information for anything other
-than getting the issue fixed for your respective distribution's users.
+
+Members of distributors-announce@kubernetes.io must share list information only
+within their teams, on a need-to-know basis to get the related issue fixed in
+their distribution. The information members and others receive from the list must
+not be made public, shared, nor even hinted at otherwise, except with the list's
+explicit approval. (For example, this means that members who do not represent a
+distribution may not use list information to give "advance warning" to anyone.)
```

Comment:
I think this is a bit confusing here, as for most people membership to distributors-announce means you work for (represent) a distribution. Maybe just call out PSC members & associates explicitly in a separate paragraph? It also applies to people brought in as part of a "fix team".

Reply:
I've pushed a rev; Do you think this is easier to read? I'm trying to make sure it adds value beyond just the special case of PSC members, but without overcomplicating things.